src/components/Reports/FourteenMonthReports/FourteenMonthReport.test.tsx

```diff
@@ -243,7 +251,7 @@ describe('TwelveMonthReport', () => {
       expect(table1MonthlyTotals[2]).toHaveTextContent('100');
       expect(table1MonthlyTotals[3]).toHaveTextContent('100');
       expect(within(tables[0]).getByTestId('overallTotal')).toHaveTextContent(
-        '400',
+        '300',
```

Comment:
Do you know why this doesn't produce `200` if we're ignoring the first and last months?